tests/config/meilisearch.yaml

```diff
@@ -40,3 +40,18 @@ meilisearch:
             class: 'Meilisearch\Bundle\Tests\Entity\DummyCustomGroups'
             enable_serializer_groups: true
             serializer_groups: ['public', 'private']
+        -   name: dynamic_settings
+            class: 'Meilisearch\Bundle\Tests\Entity\DynamicSettings'
+            settings:
+                filterableAttributes:
+                    _service: '@Meilisearch\Bundle\Tests\Integration\Fixtures\FilterableAttributes'
+                searchableAttributes:
+                    _service: '@Meilisearch\Bundle\Tests\Integration\Fixtures\SearchableAttributes'
+                stopWords:
+                    _service: '@Meilisearch\Bundle\Tests\Integration\Fixtures\StopWords'
+                synonyms:
+                    _service: '@Meilisearch\Bundle\Tests\Integration\Fixtures\Synonyms'
+
+services:
+    Meilisearch\Bundle\Tests\Integration\Fixtures\:
```

Comment:
I'm not entirely familiar with most Symfony concepts yet, so can you explain why you need that? Is it part of the feature requirements, or is it just a way to map paths to the config file?

Reply:
This way we just say that register everything in that folder as a service :slightly_smiling_face: